cosmos/airflow/graph.py

```diff
@@ -106,14 +106,20 @@ def create_test_task_metadata(
             task_args["select"] = node.resource_name
 
         extra_context = {"dbt_node_config": node.context_dict}
+        task_owner = node.owner
 
     elif render_config is not None:  # TestBehavior.AFTER_ALL
         task_args["select"] = render_config.select
         task_args["selector"] = render_config.selector
         task_args["exclude"] = render_config.exclude
+        task_owner = ""
+
+    else:
+        task_owner = ""
```

Comment:
@wornjs thanks for addressing the feedback! Could you add a test to cover this use case as well? It is looking good and we'll release this as part of Cosmos 1.6.

Reply:
I just refactored the code not to need this conditional, I believe coverage will be happy 🤞

Reply:
It worked! I'll rebase and merge, thanks for the contribution, @wornjs !